### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_ Url Rewrite- Outbound Rules_Keyboard: Focus is not visible on some of the controls in all pages

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#161]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Chrome; #Keyboard; MAS2.4.7

**Environment Details:**
URL: https://manage.iis.net/webserver/url-rewrite
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.4.7

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Url Rewrite and press enter
6. Tab till outbound rules tab and press enter
7. Verify focus is visible on all the controls or not

**Actual Result:**
Focus is not visible on some of the controls in the page

**Expected Result:**
Focus should visible on all the controls in the page

**User Impact:**
Users with disabilities finds its difficult in working through the application if focus is not properly visible on all the controls

**Note:**
Similar issue is also found in Web Server- Url Rewrite- Server Variables

**Attachments:**
![bug161_web server_url rewrite_outbound rules_focus visible1](https://user-images.githubusercontent.com/42638450/44777895-56d5d380-ab99-11e8-9d6b-d46f4434915c.png)
![bug161_web server_url rewrite_outbound rules_focus visible2](https://user-images.githubusercontent.com/42638450/44777896-56d5d380-ab99-11e8-9c60-9914732afdf7.png)
![bug161_web server_url rewrite_outbound rules_focus visible3](https://user-images.githubusercontent.com/42638450/44777898-56d5d380-ab99-11e8-9309-13ac5fee1d33.png)
![bug161_web server_url rewrite_outbound rules_focus visible4](https://user-images.githubusercontent.com/42638450/44777899-576e6a00-ab99-11e8-93f2-1403cc81d230.png)
![bug161_web server_url rewrite_server variables_focus visible5](https://user-images.githubusercontent.com/42638450/44777900-576e6a00-ab99-11e8-9b9b-bd30cf38dbf2.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
